gobblin-modules/gobblin-http/src/main/java/gobblin/r2/R2RestResponseHandler.java

```diff
@@ -42,12 +45,15 @@ public R2ResponseStatus handleResponse(Request<RestRequest> request, RestRespons
     int statusCode = response.getStatus();
     status.setStatusCode(statusCode);
 
+    EventSubmitter.Builder eventSubmitterBuilder = new EventSubmitter.Builder(metricsContext, this.getClass().getSimpleName());
```

Comment:
I just realized that properties can be overridden here. Bringing eventSubmitter out of handleResponse()...
"gobblin.http" will be a suitable namespace?